util/topgen/intermodule.py

```diff
@@ -150,21 +150,24 @@ def elab_intermodule(topcfg: OrderedDict):
                              ('struct', req_struct["struct"] + "_req"),
                              ('signame', sig_name + "_req"),
                              ('width', req_struct["width"]),
-                             ('type', req_struct["type"])]))
+                             ('type', req_struct["type"]),
```

Comment:
can we move this particular feature over to master also?

Reply:
yes, this one goes already to master ;). I'll carry it over to Bronze, once merged.